Backend/HTML/verified.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
why does Backend directory have HTML? All html files should be in frontend, right?